scala/src/main/javadocstub/JavaDocStub.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+/**
+ * Sonatype hosting requires javadoc for releasing an artifact into Maven Central.
+ * This java file is used to produce "fake" javadoc.
+ * Reference: http://central.sonatype.org/pages/requirements.html
+ */
+
+package javadocstub;
+/**
+ * @author      SparkClr Team <sparkclrdev@microsoft.com>
+ * @version     1.0                 (current version number of program)
+ * @since       2016-01-05          (the version of the package this class was first added to)
+ */
+
+public class JavaDocStub {
```

Comment:
Sorry, I am still confused. Will this class be in the doc? If so, how do other projects generate doc without this stub class? How is this class even helping with the javadoc requirement?